docs/root/faq/configuration/sni.rst

```diff
@@ -70,9 +70,9 @@ The following is a YAML example of the above requirement.
 How do I configure SNI for clusters?
 ====================================
 
-For clusters, a fixed SNI can be set in `UpstreamTlsContext <envoy_api_field_auth.UpstreamTlsContext.sni>`_.
+For clusters, a fixed SNI can be set in :ref:`UpstreamTlsContext <envoy_api_field_auth.UpstreamTlsContext.sni>`.
```

Comment:
Please merge master to get rid of this diff and fix coverage.

Reply:
done